edm4eic.yaml

```
@@ -544,3 +544,14 @@ datatypes:
      - edm4eic::RawTrackerHit rawHit        // reference to the digitized hit
     OneToManyRelations:
      - edm4hep::SimTrackerHit simHits       // reference to the simulated hits
+
+
+  edm4eic::MCParticleRecoHitAssociation:
+    Description: "Association between a TrackerHit and MCParticles. One tracker hit can be linked to multiple simulation hits hence multiple particles"
+    Author : "S. Li"
+    VectorMembers:
+      - float               weights         // weight of this association
+    OneToOneRelations:
+      - edm4eic::TrackerHit trackHit        // reference to the hit
+    OneToManyRelations:
+      - edm4hep::MCParticle mcParticles     // reference to the Monte-Carlo particle
```

I thought about this. I think the original reason for the `weight` field is that there can be multiple associations. There may be several entries in the association collection, and weight can encode the fractional contribution to the hit/cluster.
```suggestion
    Members:
      - float               weight         // weight of this association
    OneToOneRelations:
      - edm4eic::TrackerHit trackHit        // reference to the hit
      - edm4hep::MCParticle mcParticle     // reference to the Monte-Carlo particle
```
cc @sly2j @whit2333 for confirmation